distributed/cli/dask_scheduler.py

```diff
@@ -139,7 +139,7 @@ def main(
     tls_cert,
     tls_key,
     dashboard_address,
-    **kwargs
+    **kwargs,
```

Comment:
This excludes Python 3.5 .  You'll have to update your black to master, or the commit in the pre-commit config file.   :/

Reply:
Ah, didn't realize this required a non-released version of black.

Reply:
Found the lines that were confusing black and fixed them, should now work fine with the latest black release or master.